en/middleware/micrortps_throughput_test.md

```diff
@@ -0,0 +1,105 @@
+# Throughput test
```

Comment:
Title should have more context. Use First Letter capitalisation "RTPS Throughput Test". Is this in SUMMARY.md.

Cross link from other docs so it can be found.